src/main/resources/i18n/messages.properties

```diff
@@ -144,7 +144,7 @@ clan.leader=Leader
 clan.clanMembers=Members
 clan.messageLeader=Message clan leader
 clan.visitPage=Visit clan website
-chat.privateMessage.playerLeft={0} is now offline
+chat.privateMessage.playerLeft={0} is now offline and left chat
```

Comment:
Isn't that implicitly clear to the user? ;)
"Person X died and does no longer speak"